<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addwindow</class>
 <widget class="QWidget" name="addwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="logo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="searchlabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>查找</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="searcedit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searcbtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>查  找</string>
   </property>
  </widget>
  <widget class="QPushButton" name="locatedbtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sex">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QComboBox" name="age">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>62</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="closebtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hidebtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>0</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="sexlabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLabel" name="agelabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hidedetailbtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>0</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Exact_btn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>精确匹配</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="bgimage">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="namelabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="signlabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="facebtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addbtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>75</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>加好友</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="groupch" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>141</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QRadioButton" name="group1btn">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>9</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="group3btn">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>67</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="group4btn">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>96</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okbtn">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>125</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="group2btn">
    <property name="geometry">
     <rect>
      <x>29</x>
      <y>38</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>47</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="backbtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>75</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="mask">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>260</y>
     <width>47</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>bgimage</zorder>
  <zorder>logo</zorder>
  <zorder>searchlabel</zorder>
  <zorder>searcedit</zorder>
  <zorder>searcbtn</zorder>
  <zorder>locatedbtn</zorder>
  <zorder>sex</zorder>
  <zorder>age</zorder>
  <zorder>closebtn</zorder>
  <zorder>hidebtn</zorder>
  <zorder>sexlabel</zorder>
  <zorder>agelabel</zorder>
  <zorder>hidedetailbtn</zorder>
  <zorder>Exact_btn</zorder>
  <zorder>namelabel</zorder>
  <zorder>signlabel</zorder>
  <zorder>facebtn</zorder>
  <zorder>addbtn</zorder>
  <zorder>line</zorder>
  <zorder>groupch</zorder>
  <zorder>mask</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
